### PR TITLE
add rubocop

### PR DESCRIPTION
**Pull request by grosser**
fixes https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter/issues/251#issuecomment-705268717

### CODE REVIEW

**Comment by grosser** [comment]
https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter/pull/265 with just the tests to make sure it does not break both and ends up green by accident

File: .circleci/config.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ missingdeps: &missingdeps
 
 test: &test
   name: Test bundle
-  command: bundle exec rake test
+  command: bundle exec rake --trace
```
Comment (jcantrill) [inline .circleci/config.yml:20]: Why this change?

**Review by jcantrill — COMMENTED** [review]
Aside from one question it looks like formatting changes which is to be expected

File: .circleci/config.yml (hunk 1 above)
Comment (grosser) [reply .circleci/config.yml:20]: it makes it run rubocop and the test
added --trace so it's obvious if test failed or rubocop, it will show the task name

**Comment by jcantrill** [comment]
@grosser please rebase if appropriate or close

**Comment by jcantrill** [comment]
Close in favor of #278
